packages/enzyme/src/selectors.js

```diff
@@ -55,6 +64,80 @@ function safelyGenerateTokens(selector) {
   }
 }
 
+function matchAttributeSelector(node, token) {
+  const { operator, value, name } = token;
+  const nodeProps = propsOfNode(node);
+  const descriptor = Object.getOwnPropertyDescriptor(nodeProps, name);
+  if (descriptor && descriptor.get) {
+    return false;
+  }
+  const nodePropValue = nodeProps[name];
+  if (typeof nodePropValue === 'undefined') {
+    return false;
+  }
+  if (token.type === ATTRIBUTE_PRESENCE) {
+    return has(nodeProps, token.name);
+  }
+  // Only the exact value operator ("=") can match non-strings
+  if (typeof nodePropValue !== 'string' || typeof value !== 'string') {
+    if (operator !== EXACT_ATTRIBUTE_OPERATOR) {
+      return false;
+    }
+  }
+  switch (operator) {
+    /**
+     * Represents an element with the att attribute whose value is exactly "val".
+     * @example
+     * [attr="val"] matches attr="val"
+     */
+    case EXACT_ATTRIBUTE_OPERATOR:
+      return is(nodePropValue, value);
+    /**
+     * Represents an element with the att attribute whose value is a whitespace-separated
+     * list of words, one of which is exactly 
```

Comment:
this trailing space is breaking the linter

Reply:
Woops! Fixed 👌